go/vt/topo/shard.go

```diff
@@ -327,34 +327,29 @@ func (ts *Server) CreateShard(ctx context.Context, keyspace, shard string) (err
 
 // GetOrCreateShard will return the shard object, or create one if it doesn't
 // already exist. Note the shard creation is protected by a keyspace Lock.
-func (ts *Server) GetOrCreateShard(ctx context.Context, keyspace, shard, cell string) (si *ShardInfo, err error) {
-	log.Info("GetShard %s/%s", keyspace, shard)
```

Comment:
do these lines create spam? a certain amount of logging is useful.

Reply:
There's already a log line where this function is called, and if it fails, the error will tell us what went wrong. Not much additional value from these log lines.

Reply:
the logging was added here: https://github.com/vitessio/vitess/commit/cb5097e8ab43d6244ecb5ddea9b779b18ed72bf1
@harshit-gangal any objections to removing this logging?

Reply:
Offline response from @harshit-gangal that this is fine.